sgx-types/src/sig.rs

```diff
@@ -1,6 +1,22 @@
 use super::{attr::Attributes, misc::MiscSelect, utils::Padding};
 
 #[derive(Copy, Clone)]
+struct Number([u8; 384]);
```

Comment:
This is a good idea since we're using this type of array multiple times, but could use a more descriptive name. What sort of Number is it? Why is a Number 384 bytes? Etc.

Reply:
Fixed.